src/components/Center/Center.tsx

```diff
@@ -41,6 +42,8 @@ export const Center: FC<PropsWithChildren<Props>> = ({
   pr,
   pb,
   pl,
+  mt,
```

Comment:
margin-right and margin-left cannot be specified as they are used for centred alignment